os.go

```diff
@@ -190,7 +193,7 @@ func isHidden(f os.FileInfo, path string, hiddenfiles []string) bool {
 func userName(f os.FileInfo) string {
 	if stat, ok := f.Sys().(*syscall.Stat_t); ok {
 		if u, err := user.LookupId(fmt.Sprint(stat.Uid)); err == nil {
-			return fmt.Sprintf("%v ", u.Username)
+			return u.Username
```

Comment:
Perhaps we could keep appending spaces here and therefore have the same format string on Windows?

If someone really wants to get rid of the space, there's probably a terminal sequence for that ("\h"?)

Reply:
I'm not sure about this, I'm worried that it will just end up confusing users if fields end with a space, especially if you are trying to style them. I also would prefer to keep the same semantics as the existing `promptfmt` option.

Originally I just added the default in `opts.go`, and then I later realised that Windows would need a different default, but  I think it isn't really an issue.

Reply:
Yeah, you are probably right.